### PR TITLE
Add a pairing utility for Wiimotes to Cemu

### PR DESCRIPTION
**Pull request by yeah-its-gloria**
figured since Cemu can directly interact with Wiimotes this would be a handy tool to have :)

### CODE REVIEW

**Comment by Squall-Leonhart** [comment]
you should aim to have feature parity on all supported platforms

**Comment by yeah-its-gloria** [comment]
[bluez does this already](https://github.com/bluez/bluez/blob/master/plugins/wiimote.c)

**Comment by Exzap** [comment]
Seems like you left debug logging in there (`OutputDebugStringW`). Can you clean that up?

File: src/gui/PairingDialog.cpp [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+#if BOOST_OS_WINDOWS
+#include <bluetoothapis.h>
+#endif
+
+wxDECLARE_EVENT(wxEVT_PROGRESS_PAIR, wxCommandEvent);
+wxDEFINE_EVENT(wxEVT_PROGRESS_PAIR, wxCommandEvent);
+
+PairingDialog::PairingDialog(wxWindow* parent)
+    : wxDialog(parent, wxID_ANY, "Pairing...", wxDefaultPosition, wxDefaultSize, wxCAPTION | wxMINIMIZE_BOX | wxSYSTEM_MENU | wxTAB_TRAVERSAL | wxCLOSE_BOX)
```
Comment (Fs00) [inline src/gui/PairingDialog.cpp:12]: The title can be wrapped with `_(`...`)` to make it translatable

Comment (Fs00) [inline src/gui/PairingDialog.cpp:12]: The title can be wrapped with `_(`...`)` to make it translatable

File: src/gui/PairingDialog.cpp [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+{
+    auto* sizer = new wxBoxSizer(wxVERTICAL);
+    m_gauge = new wxGauge(this, wxID_ANY, 100, wxDefaultPosition, wxSize(350, 20), wxGA_HORIZONTAL);
+    m_gauge->SetValue(0);
+    sizer->Add(m_gauge, 0, wxALL | wxEXPAND, 5);
+
+    auto* rows = new wxFlexGridSizer(0, 2, 0, 0);
+    rows->AddGrowableCol(1);
+
+    m_text = new wxStaticText(this, wxID_ANY, "Searching for controllers...");
```
Comment (Fs00) [inline src/gui/PairingDialog.cpp:22]: Same here too

**Comment by Fs00** [comment]
Just a thought: wouldn't it make more sense to open the pairing utility via a button in the input settings window, instead of having an entry in the Tools menu?

**Comment by yeah-its-gloria** [comment]
I've implemented all those fixes, thanks for letting me now

**Comment by Exzap** [comment]
thanks!

**Pull request merged**